Src/AutoFixture/EmailAddressLocalPart.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Ploeh.AutoFixture
+{
+    /// <summary>
+    /// Represents the local part of the email address, defined as everything up to, but not including the @ sign.  
+    /// The requirements enforced on the local part are defined in RFC 3696.  The regular expression for validating 
+    /// the local part borrowed from Phil Haack at 
+    /// http://haacked.com/archive/2007/08/21/i-knew-how-to-validate-an-email-address-until-i.aspx/
```

Comment:
Funny article, but I'm not even sure that this regular expression is liberal enough... Here are some examples that all evaluate to false - I wonder if at least one shouldn't be allowed:

```
> regex.IsMatch "ndøh@example.com";;
val it : bool = false
> regex.IsMatch "ndöh@example.com";;
val it : bool = false
> regex.IsMatch "åhnej@example.com";;
val it : bool = false
> regex.IsMatch "ñoñó1234@server.com";;
val it : bool = false
> regex.IsMatch "f#ers.com";;
val it : bool = false
```

(Unless I did something wrong when I copy-pasted the `pattern` string - which can't *entirely* be ruled out...)